src/storage/documentService/documentService.js

```diff
@@ -47,17 +50,30 @@ const validateTtl = ttl => {
 
 const uploadDocument = async (
   document,
+  docId,
   ttl = DEFAULT_TTL,
   network = config.network
 ) => {
+  let docStoreId = docId;
+  if (docId) {
+    const response = await getDocument(docId, { cleanup: false });
+    if (!response || response.document) {
+      throw new Error("Can not find the valid queue number");
```

Comment:
I think just return status 401 and message `UUID exists` , if they get here they're probably doing something weird